src/script.js

```diff
@@ -0,0 +1,129 @@
+import './style.css';
+
+let myTasks = [];
+
+const addArray = (description, completed, index) => {
+  myTasks.push({
+    description,
+    completed: false,
+    index,
+  });
+};
+
+const getInput = (event) => {
+  if (event.key === 'Enter') {
+    const myTaskInput = document.getElementById('addtask');
+    const description = myTaskInput.value.trim();
+
+    if (description !== '') {
+      addArray(description, false, myTasks.length + 1);
+      myTaskInput.value = '';
+      displayTask(myTasks.length - 1); // Display the newly added task
+      storeItems();
+    }
+  }
+};
+
+const displayTask = (index) => {
+  const container = document.getElementById('list-items');
+
+  const item = document.createElement('li');
+  const checkbox = document.createElement('input');
+  const taskDescription = document.createElement('input');
+  const element = document.createElement('hr');
+  const kebabMenu = document.createElement('div');
+  const kebabIcon = document.createElement('span');
+  const trash = document.createElement('span');
+
+  trash.classList.add('fas', 'fa-trash', 'delete');
+
+  checkbox.type = 'checkbox';
+  checkbox.classList.add('check');
+
+  item.classList.add('list');
+  taskDescription.value = myTasks[index].description;
+  taskDescription.classList.add('disc');
+
+  kebabMenu.classList.add('kebab-menu');
+  kebabIcon.classList.add('kebab-icon');
+  kebabIcon.innerHTML = '&#8942;';
+  kebabMenu.appendChild(kebabIcon);
+
+  item.appendChild(checkbox);
+  item.appendChild(taskDescription);
+  item.appendChild(element);
+  item.appendChild(kebabMenu);
+  item.appendChild(trash);
+
+  container.appendChild(item);
+
+  taskDescription.addEventListener('keyup', (event) => {
+    if (event.key === 'Enter') {
+      const newDescription = taskDescription.value.trim();
+      if (newDescription !== '') {
+        editlist(index, newDescription);
+      } else {
+        taskDescription.value = myTasks[index].description; // Restore the previous description
+      }
+    }
+  });
+};
```

Comment:
- [ ] [OPTIONAL] I would highly recommend you to put `onChange` event listener on your `description value` when we edit it instead of enter because even after clicking enter the cursor remains on the edited description value and if we don't click enter value is not saved. Which is really confusing and took me a while to understand why the value was not being saved.